From 572688cdf5742dea52b5db54a59430e033fb0a95 Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Tue, 15 Jan 2013 12:25:42 +0200
Subject: [PATCH 065/103] bone-dvi-cape: Add DT definition for 00A2 revision

Add cape definition for the 00A2 revision of the DVI cape.

Signed-off-by: Pantelis Antoniou <panto@antoniou-consulting.com>
---
 firmware/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/firmware/Makefile b/firmware/Makefile
index 600640a..256f8d4 100644
--- a/firmware/Makefile
+++ b/firmware/Makefile
@@ -140,6 +140,7 @@ fw-shipped-$(CONFIG_YAM) += yam/1200.bin yam/9600.bin
 fw-shipped-$(CONFIG_CAPE_BEAGLEBONE) += \
 	capes/cape-bone-2g-emmc1.dtbo \
 	capes/cape-bone-dvi-00A0.dtbo capes/cape-bone-dvi-00A1.dtbo \
+		capes/cape-bone-dvi-00A2.dtbo \
 	capes/cape-bone-lcd3-00A0.dtbo
 
 # the geiger cape
-- 
1.8.1.4

